Sources/SwiftSyntax/SyntaxData.swift

```diff
@@ -194,7 +194,8 @@ struct AbsoluteRawSyntax {
 ///
 /// SyntaxData is an implementation detail, and should not be exposed to clients
 /// of SwiftSyntax.
-struct SyntaxData {
+@_spi(SyntaxData)
+public struct SyntaxData {
```

Comment:
Let's avoid exposing `SyntaxData` by rewriting `SyntaxData.contextualClassification` to `Syntax.contextualClassification`.